app/components/playlist/playlist-form.tsx

```
@@ -0,0 +1,140 @@
+    <form className="flex flex-col gap-4 p-4" onSubmit={(e) => {
+      e.preventDefault();
+      mutate();
+    }}>
+      {error && (
+        <div className="p-3 rounded bg-[--md-sys-color-error-container] text-[--md-sys-color-on-error-container]">
+          {error}
+        </div>
+      )}
+
+      <div className="flex flex-col gap-2">
+        <label>Cover Image</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)